packages/node_modules/@webex/plugin-meetings/src/scheduled-meetings/collection.js

```diff
@@ -0,0 +1,42 @@
+const ScheduledMeetingsCollection = {
+  scheduledMeetings: {},
+
+  /**
+   *
+   * @param {String} id ID of the scheduled meeting you wish to retreive
+   * @returns {MeetingInfo} returns a meeting info instance
+   */
+  get(id) {
+    return this.scheduledMeetings[id];
+  },
+
+  /**
+   *
+   * @param {String} id the id of the calendar meeting instance to add to the collection
+   * @param {Object} scheduledMeeting the full scheduled meeting
+   * @returns {undefined}
+   */
+  set(id, scheduledMeeting) {
+    this.scheduledMeetings[id] = scheduledMeeting;
+  },
+
+  /**
+   * @param {Array} scheduledMeetings array of scheduled meetings to replace current instance with
+   * @returns {undefined}
+   */
+  setAll(scheduledMeetings) {
+    this.scheduledMeetings = {};
```

Comment:
Wouldn't hurt to check that `scheduledMeetings` is indeed an `Array`

Reply:
+1